Error 'Type org.glassfish.jersey.spi.Component Provider not present' in BDK 2.2.0

## Bug Report

### Steps to Reproduce:
Generate a bot using `yo symphony 2.0` and make sure it uses BDK 2.2.0

### Actual Result:
Bot fails at startup with error message:
```
Caused by: java.lang.TypeNotPresentException: Type org.glassfish.jersey.spi.ComponentProvider not present
	at java.base/sun.reflect.generics.factory.CoreReflectionFactory.makeNamedType(CoreReflectionFactory.java:117) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.visitClassTypeSignature(Reifier.java:125) ~[na:na]
	at java.base/sun.reflect.generics.tree.ClassTypeSignature.accept(ClassTypeSignature.java:49) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.reifyTypeArguments(Reifier.java:68) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.visitClassTypeSignature(Reifier.java:138) ~[na:na]
	at java.base/sun.reflect.generics.tree.ClassTypeSignature.accept(ClassTypeSignature.java:49) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.reifyTypeArguments(Reifier.java:68) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.visitClassTypeSignature(Reifier.java:138) ~[na:na]
	at java.base/sun.reflect.generics.tree.ClassTypeSignature.accept(ClassTypeSignature.java:49) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.reifyTypeArguments(Reifier.java:68) ~[na:na]
	at java.base/sun.reflect.generics.visitor.Reifier.visitClassTypeSignature(Reifier.java:138) ~[na:na]
	at java.base/sun.reflect.generics.tree.ClassTypeSignature.accept(ClassTypeSignature.java:49) ~[na:na]
	at java.base/sun.reflect.generics.repository.ClassRepository.computeSuperclass(ClassRepository.java:104) ~[na:na]
	at java.base/sun.reflect.generics.repository.ClassRepository.getSuperclass(ClassRepository.java:86) ~[na:na]
	at java.base/java.lang.Class.getGenericSuperclass(Class.java:949) ~[na:na]
	at javax.ws.rs.core.GenericType.getTypeArgument(GenericType.java:211) ~[jakarta.ws.rs-api-2.1.6.jar:2.1.6]
	at javax.ws.rs.core.GenericType.<init>(GenericType.java:117) ~[jakarta.ws.rs-api-2.1.6.jar:2.1.6]
	at org.glassfish.jersey.client.ClientBootstrapBag$1.<init>(ClientBootstrapBag.java:55) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.ClientBootstrapBag.getComponentProviders(ClientBootstrapBag.java:55) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.ClientConfig$State.initRuntime(ClientConfig.java:447) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.internal.util.collection.Values$LazyValueImpl.get(Values.java:317) ~[jersey-common-2.33.jar:na]
	at org.glassfish.jersey.client.ClientConfig.getRuntime(ClientConfig.java:819) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.ClientRequest.getClientRuntime(ClientRequest.java:176) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.ClientRequest.getInjectionManager(ClientRequest.java:567) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.JerseyWebTarget.onBuilder(JerseyWebTarget.java:371) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:199) ~[jersey-client-2.34.jar:na]
	at org.glassfish.jersey.client.JerseyWebTarget.request(JerseyWebTarget.java:38) ~[jersey-client-2.34.jar:na]
	at com.symphony.bdk.http.jersey2.ApiClientJersey2.invokeAPI(ApiClientJersey2.java:96) ~[symphony-bdk-http-jersey2-2.2.0.jar:2.2.0]
```

As you can notice, it is using jersey-client-2.34 but jersey-common-2.33. When running `./mvnw dependency:tree` on the bot project, two jersey dependencies are in version 2.33, namely:
org.glassfish.jersey.core:jersey-common:jar:2.33:runtime
org.glassfish.jersey.ext:jersey-entity-filtering:jar:2.33:runtime

### Expected Result:
Bot working as usual
All used jersey2 dependencies should be in version 2.34

--- a/symphony-bdk-bom/build.gradle
+++ b/symphony-bdk-bom/build.gradle
@@ -60,10 +60,12 @@ dependencies {
         api 'org.openapitools:jackson-databind-nullable:0.2.1'
 
         api 'org.glassfish.jersey.core:jersey-client:2.34'
+        api 'org.glassfish.jersey.core:jersey-common:2.34'
         api 'org.glassfish.jersey.inject:jersey-hk2:2.34'
         api 'org.glassfish.jersey.media:jersey-media-multipart:2.34'
         api 'org.glassfish.jersey.media:jersey-media-json-jackson:2.34'
         api 'org.glassfish.jersey.connectors:jersey-apache-connector:2.34'
+        api 'org.glassfish.jersey.ext:jersey-entity-filtering:2.34'
 
         api 'org.projectreactor:reactor-spring:1.0.1.RELEASE'
 

